.github/workflows/publish-sphinx.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
     steps:
     - name: Checkout
-      uses: actions/checkout@master
+      uses: actions/checkout@main
```

Comment:
I see, refers to default branch pf this repo: https://github.com/actions/checkout
Thanks.